### PR TITLE
Add WSL guide for building shards

### PR DESCRIPTION
**Pull request by alebaffa**
Added few additional lines after my experience on building Shards on WSL (Ubuntu).

### CODE REVIEW

**Review by satendrapalsingh — APPROVED** [review]
Looks good to me!

File: docs/docs/contribute/code/building-shards.md [hunk 1]
```diff
@@ -195,7 +226,11 @@ You need to run the following two commands every time you want to build the proj
 Configure the build with `cmake`,
 
 ```
+# Windows
 cmake -G Ninja -DCMAKE_BUILD_TYPE=Debug -DCMAKE_EXPORT_COMPILE_COMMANDS=1 ..
```
Comment (sinkingsugar) [inline docs/docs/contribute/code/building-shards.md:230]: `CMAKE_EXPORT_COMPILE_COMMANDS=1` is not necessary in either.

Comment (alebaffa) [reply docs/docs/contribute/code/building-shards.md:230]: yeah, I was wondering about that...let me remove it.

**Review by sinkingsugar — APPROVED** [review]

**Comment by alebaffa** [comment]
May I merge this or better to let someone who work more closely with Shards do it?

**Pull request merged**